submissions/kasionio/oop-exercise/index.js

```diff
@@ -0,0 +1,108 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/kasionio/a-tiny-JS-world/tree/gh-pages
+   Web app: https://kasionio.github.io/a-tiny-JS-world
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(species, name, gender, legs, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.saying = saying;
+      this.friends = [];
+  }
+
+  addFriends(...friendsInstances) {
+      this.friends.push(...friendsInstances);
+  }
+
+  getFriendsNames() {
+      if (this.friends.length > 0) {
+          return this.friends.map(friend => friend.name)
+      } else {
+          return this.friends = 'forever alone...'
+      }
```

Comment:
1. `return this.friends = 'forever alone...'`: try the following:
```
print(woman.getOutput());
print(woman.getOutput());
```
You will see the issue caused by this fragment.

2. The entire method body is a typical case for the ternary operator. Check MDN for details.
3. Will you take a challenge to master the ternary?

Reply:
1. Everything looks as expected to me now.
It prints `PRINT: species: human; name: Kate; gender: female; legs: 2; saying: Merry Christmas!; friends: Arnold,Bobby` every time. 
2. Change to ternary